sdk/tables/data-tables/src/TableClient.ts

```diff
@@ -268,6 +269,25 @@ export class TableClient {
       generatedClient.pipeline.addPolicy(apiVersionPolicy(options.version));
     }
 
+    if (options.georedundantRetryOptions) {
+      const maxRetries =
+        (options.georedundantRetryOptions.maxRetries ?? 3) *
+        Math.max(
+          options.georedundantRetryOptions.readHosts?.length ?? 1,
+          options.georedundantRetryOptions.writeHosts?.length ?? 1
+        );
+      generatedClient.pipeline.removePolicy({ phase: "Retry" });
+      generatedClient.pipeline.addPolicy(
+        retryPolicy(
+          [throttlingRetryStrategy(), georedundantRetryStrategy(options.georedundantRetryOptions)],
```

Comment:
since this omits `exponentialRetryStrategy` we won't retry in the case of system errors anymore? It also doesn't seem like georedundant retries on a 408 either.

Reply:
Good catch! I'll go ahead and add handling for system errors once we have the design locked down. As for 408s, should we retry the same URL in that case? If so, I think just adding the exponential policy after this should work. I think the factory/injection discussion below addresses this too.

Reply:
Yeah I think you could probably just add in the exponential strategy on top (potentially configuring it to ignore some things)

Reply:
This should be fixed now that it defers to `exponentialRetryStrategy` for 408s and system errors.